src/preload/rendererBridge/index.ts

```diff
@@ -12,7 +12,7 @@ const api = {
 	},
 };
 
-if (preloadData.contextIsolation) {
+if (!!preloadData.windowOptions.webPreferences.contextIsolation) {

```

Comment:
This should be optionally chained to prevent errors with other Electron apps.

`!!preloadData.windowOptions.webPreferences?.contextIsolation`